src/MicrophysicsNonEq.jl

```diff
@@ -101,18 +103,38 @@ function conv_q_vap_to_q_liq_ice_MM2015(
     end
 end
 function conv_q_vap_to_q_liq_ice_MM2015(
-    (; τ_relax)::CMP.CloudIce,
+    (; τ_relax)::CMP.CloudLiquid{FT},
     tps::TDP.ThermodynamicsParameters,
     q::TD.PhasePartition,
     ρ,
     T,
 )
+
+    override_file = Dict(
+        "condensation_evaporation_timescale" =>
+            Dict("value" => τ, "type" => "float"),
+    )
+    liquid_toml_dict = CP.create_toml_dict(FT; override_file)
+    liquid = CMP.CloudLiquid(liquid_toml_dict)
+
+    conv_q_vap_to_q_liq_ice_MM2015(liquid, tps, q.tot, q.liq, q.ice, ρ, T)
+
+end
+function conv_q_vap_to_q_liq_ice_MM2015(
```

Comment:
Could you provide wrapper functions that still accept `PhasePartition` as argument for both liquid and ice?